app/src/main/java/lt/vilnius/tvarkau/entity/Problem.kt

```diff
@@ -0,0 +1,52 @@
+package lt.vilnius.tvarkau.entity
+
+import com.google.android.gms.maps.model.LatLng
+import com.google.gson.annotations.SerializedName
+import org.parceler.Parcel
+import org.threeten.bp.LocalDateTime
+
+@Parcel
+data class Problem(
```

Comment:
I would prefer to have separate classes for `Problem` and `ProblemResponseDto` with `ProblemMapper`. In this way inside `Problem` class we wouldn't have unneeded fields for example `typeName` and `type` would become `type` enum, `entryDate` and `reportDate` would become `entryDate`, `photo` and `thumbnail` would be `photos` ...